cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -124,8 +124,11 @@ bool CobTwistController::initialize()
 	///initialize configuration control solver
 	p_fksolver_vel_ = new KDL::ChainFkSolverVel_recursive(chain_);	//used for debugging
 	//p_iksolver_vel_ = new KDL::ChainIkSolverVel_pinv(chain_, 0.001, 5);
-	p_augmented_solver_ = new augmented_solver(chain_, 0.001, 5);
+	p_augmented_solver_ = new AugmentedSolver(chain_, 0.001, 5);
 	
+	// Before setting up dynamic_reconfigure server: init AugmentedSolverParams with default values
+	this->initAugmentedSolverParams();
```

Comment:
I think, we can skip this as DynamicReconfigure uses either its default values from the cfg or those are overwritten by parameters on the ParameterServer (loaded from a yaml)